test/support/test_helpers.ex

```diff
@@ -3104,7 +3104,8 @@ defmodule Oli.TestHelpers do
     insert(:published_resource, %{
       publication: publication,
       resource: page_revision.resource,
-      revision: page_revision
+      revision: page_revision,
+      author: insert(:author, email: "some_email@email.com")
```

Comment:
I added this part because I found that this can also be problematic when trying to insert a published resource with an author sharing the same email from the factory. A better approach would be to generate some random number to be added to the email.